packages/components/package.json

```diff
@@ -70,15 +70,11 @@
     "uuid": "^3.4.0"
   },
   "devDependencies": {
-    "@storybook/addon-a11y": "^6.4.19",
-    "@storybook/addon-actions": "^6.4.19",
```

Comment:
this one is required by stories, we should keep it

Reply:
I find it hard if we upgrade sb in the future, libs and apps have to know that starting this version of talend/scripts we need to sync our addon-action version to a compatible version.
Just an idea: what about exposing that from @talend/scripts-config-storybook-lib ? Not sure it's a good one

Reply:
In fact, actions are already in essentials package